vhdbuilder/packer/install-dependencies.sh

```diff
@@ -296,8 +296,8 @@ AMD64_ONLY_CNI_VERSIONS="
 "
 #Please add new version (>=1.4.12) in this section in order that it can be pulled by both AMD64/ARM64 vhd
 MULTI_ARCH_VNET_CNI_VERSIONS="
-1.4.22
```

Comment:
azure/swift-cni 1.4.22 and overlay-cni 1.4.27 is currently used in aks-rp side. I think we should cache this version in AKS VHD. 

![image](https://user-images.githubusercontent.com/39974812/185976189-39015450-09c7-406c-be82-5c2a4f72eb12.png)